install_yq.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
 download_yq() {
-  echo "⬇️ mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 to yq"
-  wget --no-verbose -O yq https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64
+  echo "⬇️ mikefarah/yq/releases/download/v4.40.5/yq_linux_amd64 to yq"
+  wget --no-verbose -O yq https://github.com/mikefarah/yq/releases/download/v4.40.5/yq_linux_amd64
```

Comment:
finally realized why i wasn't able to download more recent version - they changed versioning from `3` to `v4` since 4.0.0

i'm going to fix tests now

Reply:
This is actually pinned to an older version on purpose because the newer version was astronomically slower for some reason.  They may have fixed it, but be aware, the CLI interface changes frequently, so I doubt it'll work without further changes.

Personally, I wouldn't change this.